src/xml/node.cr

```diff
@@ -172,27 +172,27 @@ struct XML::Node
   def inspect(io : IO) : Nil
     io << "#<XML::"
     case type
-    when XML::Type::ELEMENT_NODE       then io << "Element"
-    when XML::Type::ATTRIBUTE_NODE     then io << "Attribute"
-    when XML::Type::TEXT_NODE          then io << "Text"
-    when XML::Type::CDATA_SECTION_NODE then io << "CData"
-    when XML::Type::ENTITY_REF_NODE    then io << "EntityRef"
-    when XML::Type::ENTITY_NODE        then io << "Entity"
-    when XML::Type::PI_NODE            then io << "ProcessingInstruction"
-    when XML::Type::COMMENT_NODE       then io << "Comment"
-    when XML::Type::DOCUMENT_NODE      then io << "Document"
-    when XML::Type::DOCUMENT_TYPE_NODE then io << "DocumentType"
-    when XML::Type::DOCUMENT_FRAG_NODE then io << "DocumentFragment"
-    when XML::Type::NOTATION_NODE      then io << "Notation"
-    when XML::Type::HTML_DOCUMENT_NODE then io << "HTMLDocument"
-    when XML::Type::DTD_NODE           then io << "DTD"
-    when XML::Type::ELEMENT_DECL       then io << "Element"
-    when XML::Type::ATTRIBUTE_DECL     then io << "AttributeDecl"
-    when XML::Type::ENTITY_DECL        then io << "EntityDecl"
-    when XML::Type::NAMESPACE_DECL     then io << "NamespaceDecl"
-    when XML::Type::XINCLUDE_START     then io << "XIncludeStart"
-    when XML::Type::XINCLUDE_END       then io << "XIncludeEnd"
-    when XML::Type::DOCB_DOCUMENT_NODE then io << "DOCBDocument"
+    when XML::Node::Type::ELEMENT_NODE       then io << "Element"
```

Comment:
Why not use shorter `.element_node?` notation?

ditto elsewhere

Reply:
I stated doing that but then kept it simple. It was easier to do the rename like this and from a user perspective there's no difference. I didn't want to introduce extra changes here. But we could do this in a separate PR.